hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -626,8 +626,8 @@ cdef class PureMarketMakingStrategy(StrategyBase):
         bid_order_id = self.c_buy_with_specific_market(
                     maker_market,
                     market_pair.maker_symbol,
-                    float(self.order_size),
-                    float(place_bid_price),
+                    self.order_size,
+                    place_bid_price,
```

Comment:
WTF!!

Reply:
`place_bid_price` has become a `Decimal` here and @veecos2  you're feeding it to `c_buy_with_specific_market()`. This is a type error.